src/app/Topology/Actions/NodeActions.tsx

```diff
@@ -94,7 +94,8 @@ export interface NodeAction {
   action?: NodeActionFunction;
   title?: React.ReactNode;
   isSeparator?: boolean;
-  blackList?: NodeType[];
+  include?: NodeType[]; // Empty means all
+  blackList?: NodeType[]; // Empty means none
```

Comment:
https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language

Let's call this something like 'blockList' or 'denyList'

Reply:
oh sorry! thanks for pointing that out...